addons/explosives/functions/fnc_allowDefuse.sqf

```diff
@@ -0,0 +1,31 @@
+#include "script_component.hpp"
+/*
+ * Author: Walthzer
+ * Excludes a passed mine from the dynamic defuse action.
+ *
+ * Arguments:
+ * 0: Mine <OBJECT>
+ * 1: Allow Ace defusal <BOOL>
+ *
+ * Return Value:
+ * Succes <BOOLEAN>
+ *
+ * Example:
+ * [_mine] call ace_explosives_fnc_allowDefuse
+ *
+ * Public: Yes
+ */
+
+params [["_mine", objNull, [objNull]], ["_allow", true, [true]]];
+TRACE_1("params",_mine,_allow);
+
+if (!(_mine in allMines)) exitWith {false};
+    
+if (_allow) then {
+    if ([_mine] call FUNC(isAllowedDefuse)) exitWith {false};
```

Comment:
Won't this just exit the `if` scope above instead of the entire function?

Reply:
Hmm, yeah forgot about that.

Reply:
Yup, you're right. Fixing now